examples/ttn-otaa/ttn-otaa.ino

```diff
@@ -31,8 +31,8 @@
  *
  *******************************************************************************/
 
-#include <lmic.h>
-#include <hal/hal.h>
+#include "../../src/lmic.h"
+#include "../../src/hal/hal.h"
```

Comment:
Oddly, this change causes a compile failure in CI. Since this is an example, I think it probably does not want the pointer changed.